### DISCUSSION TITLE
StoryViewer Font Size

### DISCUSSION THREAD

**Discussion by Alex-Toucan** [ATProductsLLC/atproducts.xyz#340] [Ideas]
Thanks @VenenJean!
![Screenshot_20241104_073117_Discord](https://github.com/user-attachments/assets/7b7abfa9-be77-4254-8cb3-c44f08985816)
![image-21](https://github.com/user-attachments/assets/d9d863dd-a988-4a46-9c6e-d72315d7c78f)
![image-49](https://github.com/user-attachments/assets/7f58f683-143b-49a8-8663-12c0b40c4622)

**Comment by ghost** [comment]
my pleasure ;)
